migrations/9_ribbon_eth_put.js

```diff
@@ -0,0 +1,71 @@
+const RibbonThetaVault = artifacts.require("RibbonThetaVault");
+const AdminUpgradeabilityProxy = artifacts.require("AdminUpgradeabilityProxy");
+const ProtocolAdapterLib = artifacts.require("ProtocolAdapter");
+const { encodeCall } = require("@openzeppelin/upgrades");
+const { ethers, BigNumber } = require("ethers");
+const { parseEther } = ethers.utils;
+
+const {
+  updateDeployedAddresses,
+} = require("../scripts/helpers/updateDeployedAddresses");
+const ACCOUNTS = require("../constants/accounts.json");
+const DEPLOYMENTS = require("../constants/deployments.json");
+const EXTERNAL_ADDRESSES = require("../constants/externalAddresses.json");
+
+module.exports = async function (deployer, network) {
+  const networkLookup = network.replace("-fork", "");
+  const { admin, owner } = ACCOUNTS[networkLookup];
+
+  await ProtocolAdapterLib.deployed();
+
+  await deployer.link(ProtocolAdapterLib, RibbonThetaVault);
+
+  // Deploying the logic contract
+  await deployer.deploy(
+    RibbonThetaVault,
+    EXTERNAL_ADDRESSES[networkLookup].assets.weth,
+    DEPLOYMENTS[networkLookup].RibbonFactory,
+    EXTERNAL_ADDRESSES[networkLookup].assets.weth,
+    EXTERNAL_ADDRESSES[networkLookup].assets.usdc,
+    EXTERNAL_ADDRESSES[networkLookup].airswapSwap,
+    18,
+    // WETH: 10**18, 10**10 0.0000001
+    // WBTC: 0.000001
+    BigNumber.from("10").pow(BigNumber.from("10")).toString(), // WBTC 10**3
```

Comment:
The `minimumSupply` parameter refers to the amount of `asset`, which in the case of put vaults is USDC. We need to set it to a number smaller than `10**6`, e.g. `10**3`